utils/cloudevent.go

```diff
@@ -0,0 +1,108 @@
+package utils
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"strings"
+	"time"
+)
+
+type CloudEvent struct {
+	CloudEventsVersion string      `json:"cloudEventsVersion"`
+	EventID            string      `json:"eventID"`
+	Source             string      `json:"source"`
+	EventType          string      `json:"eventType"`
+	EventTypeVersion   string      `json:"eventTypeVersion"`
+	EventTime          time.Time   `json:"eventTime"` // TODO: ensure rfc3339 format
```

Comment:
can delete TODO , this is the case https://golang.org/pkg/time/#Time.UnmarshalJSON

Reply:
done